sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1677,25 +1677,23 @@ class PlanParserSuite extends AnalysisTest {
         ScriptInputOutputSchema(List.empty, List.empty, None, None,
           List.empty, List.empty, None, None, false)))
 
-    // verify with ROW FORMAT DELIMETED
+    // verify with ROW FORMAT DELIMITED
     assertEqual(
-      """
-        |SELECT TRANSFORM(a, b, c)
-        |  ROW FORMAT DELIMITED
-        |  FIELDS TERMINATED BY '\t'
-        |  COLLECTION ITEMS TERMINATED BY '\u0002'
-        |  MAP KEYS TERMINATED BY '\u0003'
-        |  LINES TERMINATED BY '\n'
-        |  NULL DEFINED AS 'null'
-        |  USING 'cat' AS (a, b, c)
-        |  ROW FORMAT DELIMITED
-        |  FIELDS TERMINATED BY '\t'
-        |  COLLECTION ITEMS TERMINATED BY '\u0004'
-        |  MAP KEYS TERMINATED BY '\u0005'
-        |  LINES TERMINATED BY '\n'
-        |  NULL DEFINED AS 'NULL'
-        |FROM testData
-      """.stripMargin,
+      "SELECT TRANSFORM(a, b, c)\n" +
```

Comment:
Although this writing may seem ugly, it seems better than the previous version, haha